server/data_structures/internal_datafacade.hpp

```diff
@@ -447,13 +448,15 @@ template <class EdgeDataT> class InternalDataFacade final : public BaseDataFacad
         }
 
         return m_static_rtree->IncrementalFindPhantomNodeForCoordinate(
-            input_coordinate, resulting_phantom_node_vector, number_of_results);
+            input_coordinate, resulting_phantom_node_vector, number_of_results, 1100, bearing, range);
```

Comment:
Arg, I cringed as I typed that, I meant to re-order those parameters so it didn't need to be specified.  I'll fix this.